### PR TITLE
fix: course id and typos

### PR DESCRIPTION
**Pull request by 0-BSCode**
_Any change needs to be discussed before proceeding. Failure to do so may result in the rejection of the pull request._

## Problem Context

Explain the **details** for making this change. What existing problem does the pull request solve?

- Typo in README where **repository** was spelled as **repostiory**
- Request to change filename to leave out name of owner

## Solution

Explain how you addressed the Problem Context outlined previously.

- Fixed typo
- Edited filename

## Closing Issues

Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such).

### CODE REVIEW

**Comment by Luzefiru** [comment]
Is there any particular reason author is omitted from the file names?

I initially added them because we might run into similar reviewer titles, so the name sort of acted like a unique identifier together with the full file name.

We're in dire need of a naming convention for files so that they don't conflict with each other.

Do you have any suggestions for that, @0-BSCode?

**Comment by 0-BSCode** [comment]
It was mentioned by Sab and Wayne to remove the name of the author from the file (not sure why). Maybe we can use USC ID instead? Wdyt @Luzefiru

**Comment by thesabrinahyap** [comment]
Hello! It's not about removing the name of the author but instead, change the title of the folders. It was cis 1102 but should be 1201. Thanks!

**Comment by Luzefiru** [comment]
> It was mentioned by Sab and Wayne to remove the name of the author from the file (not sure why). 

I clarified with Sab, and she said that the issue was that there was files with CIS1102 instead of CIS1201. She's fine with having name attribution on the files.

> Maybe we can use USC ID instead? Wdyt @Luzefiru

If authors don't like attribution to keep the material anonymous or etc, we'd better not use USC ID since that defeats the purpose.

I'm thinking we can follow the format `YYYYMMDD-slug-of-header`, where
- `YYYYMMDD` is the time the note/file is created.
- `slug-of-header` is the first heading or title found in the file uploaded.

What do you think?

**Comment by 0-BSCode** [comment]
ohh I see. I'll make the changes

**Comment by 0-BSCode** [comment]
@Luzefiru let's just keep the names as is for now with the student name being like the unique identifier. Some files don't indicate when they were created

I'll just rename the folder to its correct name indicated as indicated by @thesabrinahyap

File: docs/courses/cis1201/resources/index.md [hunk 1]
```diff
@@ -10,8 +10,8 @@ title: 'Resources'
 
 ## Practice Exams
 
-- [Wayne Dayata's CIS1102 Finals Mock Exam, part 1](./files/wayne-cis1201-final-mock-exam-test1.pdf) and then [Wayne's CIS1102 Finals Mock Exam, part 2](./files/wayne-cis1201-final-mock-exam-test2.pdf)
-  - [Wayne's CIS1102 Finals Mock Exam Answer Key](./files/wayne-cis1201-final-mock-exam-ans.pdf)
-  - [Wayne's CIS110 Finals Mock Exam (Retake Version)](wayne-cis1201-final-mock-exam-retake.pdf)
+- [Wayne Dayata's CIS12021 Finals Mock Exam, part 1](./files/wayne-cis1201-final-mock-exam-test1.pdf) and then [Wayne's CIS1102 Finals Mock Exam, part 2](./files/wayne-cis1201-final-mock-exam-test2.pdf)
```
Comment (Luzefiru) [inline docs/courses/cis1201/resources/index.md]: Please change `CIS12021` to `CIS1201`.

**Review by Luzefiru — CHANGES_REQUESTED** [review]
LGTM except 1 typo. I'll merge once it's resolved. Thank you, Bry!

File: docs/courses/cis1201/resources/index.md (hunk 1 above)
Comment (0-BSCode) [reply docs/courses/cis1201/resources/index.md]: Oops. Nice catch!

**Review by Luzefiru — APPROVED** [review]

**Pull request merged**